src/actions/ScamWarningActions.tsx

```diff
@@ -23,7 +23,7 @@ export const triggerScamWarningModal = async (disklet: Disklet) => {
 
       return (
         <ConfirmContinueModal bridge={bridge} title={lstrings.warning_scam_title}>
-          <Paragraph isWarning>{warningMessage}</Paragraph>
+          <Paragraph fontType="warning">{warningMessage}</Paragraph>
```

Comment:
The `Paragraph` component doesn't take a `fontType` prop yet. Did this get into the wrong commit?